### PR TITLE
Removed Duplicate Panel Information

### PR DESCRIPTION
**Pull request by Steedie**
## What
Removed duplicate information that appears on both the building panel, and tile panel.

### Before
image

### After
image

## Why
The issue that the information is repeated in these two panels was pointed out during the last playtest. This solves that problem, and helps reduce the overall screenspace the UI on the right is taking up.

### CODE REVIEW

**Comment by Steedie** [comment]
I noticed while working on this change that the description font sizes are different in these two panels:
image

This is how it would look if they were using the same `className`
image

**Comment by chrisfarms** [comment]
> I noticed while working on this change that the description font sizes are different in these two panels...

yeah I like the green version 👍

File: frontend/src/components/panels/tile-info-panel.tsx [hunk 1]
```diff
@@ -228,17 +228,8 @@ const TileBuilding: FunctionComponent<TileBuildingProps> = ({ building, kinds, w
...
-                </span>
-                {gooRatesInNameOrder.map((goo) => (
-                    <span key={goo?.name} className="label" style={{ width: '30%' }}>
-                        <strong>{goo?.name.toUpperCase().slice(0, 1)}:</strong>{' '}
-                        {`${Math.floor((goo?.gooPerSec || 0) * 3600)}/h`}
-                    </span>
-                ))}
                 {author && (
-                    <span className="label">
+                    <span className="label" style={{ marginTop: '2rem' }}>
```
[the remark this answers is not in the document]
Comment (Steedie) [reply frontend/src/components/panels/tile-info-panel.tsx:232]: Sounds good, implementing that now.

[the remark this answers is not in the document]
Comment (Steedie) [reply frontend/src/components/panels/tile-info-panel.tsx]: Changed line 160 to
```js
const author = world?.players.find((p) => p.id === building?.kind?.owner?.id) ?? {
        addr: 'unknown',
        name: { value: 'unknown' },
    };
```
to stop line 229 from getting confused.

**Review by chrisfarms — APPROVED** [review]
LGTM

**Pull request merged**